docs/security.md

```diff
@@ -63,6 +63,11 @@ Sourcegraph on-premise allows you to have the most control over the deployment a
 - Instances are updated monthly, and are actively maintained to keep the service up and healthy.
 - Managed instances are monitored 24x7 and incidents managed in the same way as our Sourcegraph Cloud deployment.
 
+## Shared security responsibility model for sourcegraph.com and managed instances.
+
+- Sourcegraph handles the security of the applications, the systems they run on, and the environments those systems are hosted within.
+- As a customer you are responsible for the proper management of information on your account. You have to control the users, access to your data, and what extensions you install and trust. Finally, you are responsible for ensuring your company is meeting compliance requirements.
```

Comment:
Would it be worthwhile to expand on what we mean by "proper management of information on your account"?  I prefer be as explicit as possible.

Reply:
sounds good, do you have some initial items we should include?

Reply:
I was thinking of some examples for clarity such as ensuring proper access to your Access Tokens, ensuring Code Host Connections and Linked Repositories are correct.  Also that they understand the security impact that these various items have on the integrity and confidentiality of their code.  

There must be a better way to word that.

Does that make sense?